### PR TITLE
test: Review from QAs

### PR DESCRIPTION
**Pull request by tuanphamcybozu**

### CODE REVIEW

File: features/export_for_review.feature [hunk 1]
```diff
@@ -0,0 +1,475 @@
...
+    And Load username and password of the app "app_for_export" with exact permissions "view" as env vars: "USERNAME" and "PASSWORD"
+    When I run the command with args "record export --base-url $$TEST_KINTONE_BASE_URL --app $APP_ID --username $USERNAME -p $PASSWORD"
+    Then I should get the exit code is zero
+    And The output message should match with the data below:
+      | Record_number | Text  | Number |
+      | \d+           | Alice | 10     |
+      | \d+           | Bob   | 20     |
+      | \d+           | Jenny | 30     |
+
+  Scenario: CliKintoneTest-90 Should return the record contents and download attachments successfully with an existing directory.
```
Comment (hisasami) [inline features/export_for_review.feature:160]: download attachments successfully **to** an existing directory
This sounds closer to the behavior in my opinion🧐

File: features/export_for_review.feature [hunk 2]
```diff
@@ -0,0 +1,475 @@
...
+    Then I should get the exit code is zero
+    # [\/\\\\] is used to match both Windows and Unix path separators
+    And The output message should match with the data below:
+      | Record_number      | Text  | Attachment                                     |
+      | $RECORD_NUMBERS[0] | Alice | Attachment-$RECORD_NUMBERS[0][\/\\\\]file1.txt |
+    And The directory "exported-attachments" should contain files as below:
+      | FilePath                      | FileName  | Content |
+      | Attachment-$RECORD_NUMBERS[0] | file1.txt | 123     |
+
+  Scenario: CliKintoneTest-91 Should return the record contents and download attachments successfully with a non-existent directory.
```
Comment (hisasami) [inline features/export_for_review.feature:180]: successfully **to** a non-existent directory
Here too🧐

File: features/export_for_review.feature [hunk 3]
```diff
@@ -0,0 +1,475 @@
...
+    When I run the command with args "record export --base-url $$TEST_KINTONE_BASE_URL --app $APP_ID --api-token $API_TOKEN --attachments-dir new-directory"
+    Then I should get the exit code is zero
+    And The output message should match with the data below:
+      | Record_number      | Text  | Attachment                                     |
+      | $RECORD_NUMBERS[0] | Alice | Attachment-$RECORD_NUMBERS[0][\/\\\\]file1.txt |
+    And The directory "new-directory" should contain files as below:
+      | FilePath                      | FileName  | Content |
+      | Attachment-$RECORD_NUMBERS[0] | file1.txt | 123     |
+
+  Scenario: CliKintoneTest-92 Should return the record contents and download attachments successfully with attachments in different records.
```
Comment (hisasami) [inline features/export_for_review.feature:198]: When we designed this (and CliKintoneTest-93)tests from QA's point of view, we tried to differentiate txt files and image files.
Do you think tests with image downloads are necessary from code-wise point of view?

File: features/export_for_review.feature [hunk 4]
```diff
@@ -0,0 +1,475 @@
...
+      | \d+           | Bob   | 20     |
+      | \d+           | Jenny | 30     |
+
+  Scenario: CliKintoneTest-98 Should return the record contents with valid --order-by option.
+    Given The app "app_for_export" has no records
+    And The app "app_for_export" has some records as below:
+      | Text  | Number |
+      | Alice | 10     |
+      | Bob   | 20     |
+      | Jenny | 30     |
```
Comment (hisasami) [inline features/export_for_review.feature:307]: This is just recommendation, but for this kind of test, we usually use a random order like below:
      | Text  | Number |
      | Alice | **20**     |
      | Bob   | **10**     |
      | Jenny | 30     |

File: features/delete_for_review.feature [hunk 5]
```diff
@@ -0,0 +1,360 @@
+@isolated
+Feature: cli-kintone delete command
+
+  Scenario: CliKintoneTest-126 Should return the error message when the user has no privilege to delete records.
```
Comment (hisasami) [inline features/delete_for_review.feature:4]: It seems that token permissions are used to verify (not user privilege).
How about updating the scenario like this?
`when the user has no privilege to delete records`
↓
`with an API Token without Delete permission`

**Comment by tuanphamcybozu** [comment]
This draft PR is done for review.
So, I close this PR.
Ref: https://bozuman.cybozu.com/k/#/space/5275/thread/70153/4888264/9441335

**Pull request closed without merging**